diff --git a/node_modules/resize-observer-polyfill/src/index.d.ts b/node_modules/resize-observer-polyfill/src/index.d.ts
index eeb84ff..2a602df 100644
--- a/node_modules/resize-observer-polyfill/src/index.d.ts
+++ b/node_modules/resize-observer-polyfill/src/index.d.ts
@@ -7,6 +7,7 @@ interface DOMRectReadOnly {
     readonly right: number;
     readonly bottom: number;
     readonly left: number;
+    toJSON(): any;
 }
 
 declare global {
